src/recipes/aligned-header.scss

```diff
@@ -0,0 +1,20 @@
+%aligned-header {
+  // Public API (allowed to be overridden)
```

Comment:
Note: This is not included in the package, just used in the documentation. The docs show it as an example.